frontend/src/hooks/usePersonnel.tsx

```diff
@@ -28,12 +29,17 @@ const usePersonnel = (): {
         const response =
           program && (await AxiosPrivate.get(`/${program}/${profileId}`));
         response && setPersonnel(response.data);
+        setRefetch(false);
       } catch (e) {
         console.log(e);
       }
     })();
   }, [refetch]);
 
+  const fetch = () => {
+    setRefetch(true);
```

Comment:
If we call !refetch instead of tracking whether fetch is currently true/false then we can reliably trigger a re-fetch each time that we call setRefetch(!refetch). I'm not sure I understand the intention with explicitly setting true/false (but it is very possible that I am missing something!)

Reply:
(sorry this is a duplicate comment to the one above)